lib/resources/arrays.dart

```diff
@@ -5,6 +5,7 @@ enum BottomPickerType {
   time,
   dateTime,
   rangeDateTime,
+  rangeTime,
```

Comment:
`rangeTIme` enum item is not used in the project code

Reply:
重点词汇
101/5000
计算机
At first I meant to write rangeTime, but I didn't use it because of the ambiguous semantics of rangeDateTime and rangeTime. I thought about it now and decided to change rangeDateTime to rangeDate

Reply:
At first I meant to write rangeTime, but I didn't use it because of the ambiguous semantics of rangeDateTime and rangeTime. I thought about it now and decided to change rangeDateTime to rangeDate